device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-Q64/sai.profile.j2

```diff
@@ -0,0 +1,14 @@
+{# Get sai.profile based on switch_role #}
+{%- if DEVICE_METADATA is defined and DEVICE_METADATA['localhost'] is defined and DEVICE_METADATA['localhost']['subtype'] is defined -%}
+{%-     set switch_subtype = DEVICE_METADATA['localhost']['subtype'] -%}
+{%-     if 'dualtor' in switch_subtype.lower() %}
+{%         set sai_profile_contents = 'SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/th2-a7260cx3-64-64x40G-dual-tor.config.bcm' -%}
+{%-     else %}
+{%-         set sai_profile_contents = 'SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/th2-a7260cx3-64-64x40G.config.bcm' -%}
+{%-     endif %}
+{%- else %}
+{%-     set sai_profile_contents = 'SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/th2-a7260cx3-64-64x40G.config.bcm' -%}
```

Comment:
it will make more sense to template the sai.profile, so that you won't have two largely duplicated sai.profiles.

Reply:
Will try a different way to avoid having a new large config.bcm file.  Will add new logic to automatically append the new SOC properties.
Will abandon this PR and raise a new one once the new implementation is complete.